Wrappers.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include "Classroom.h"
+#include "RecreationCenter.h"
+#include "Gym.h"
+#include "Loan.h"
+#include "LoanManager.h"
```

Comment:
Go through all of the types written below.  If you're not using them (meaning, not accessing a member variable, member function, or nested type), you can forward declare them.  Include these header files where they're used: wherever you're accessing a member variable, member function, or nested type.
- [ ] revise, Don't include what you Don't use (or Only include what you Do use)